support/Python/Schedule.py

```diff
@@ -411,13 +420,43 @@ def schedule(input_file_template: Union[str, Path],
                     "  -p lev=1...3 --run-dir 'Lev{{ lev }}'"),
         force=force)
 
+    # Validate input file
+    metadata, input_file = yaml.safe_load_all(rendered_input_file)
+    # - Validate input file syntax
+    validate_process = subprocess.run([
+        executable, "--check-options", "--input-file",
+        input_file_path.resolve()
+    ],
+                                      cwd=run_dir,
+                                      capture_output=True,
+                                      text=True)
+    try:
+        validate_process.check_returncode()
+    except subprocess.CalledProcessError as err:
+        raise RuntimeError(f"Invalid input file '{input_file_path}':\n\n" +
+                           validate_process.stderr) from err
+    # - If the input file may request resubmissions, make sure we have a
+    #   segments directory
+    wallclock_exit_phase_change = find_phase_change(
+        "CheckpointAndExitAfterWallclock", input_file)
+    if wallclock_exit_phase_change is not None and not segments_dir:
+        raise ValueError(
+            "Found 'CheckpointAndExitAfterWallclock' in the input file but "
+            "no 'segments_dir' ('--segments-dir' / '-o') is set. "
```

Comment:
`-o` is for run dir. `-O` is for segments dir

Reply:
oops